ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -132,6 +132,13 @@ defaults <- node_group <- node_group.node <- node
 
       # Node management IP address | Optional.
       mgmt_ip: < IPv4_address/Mask >
+
+      # The following variable SHOULD be set to the MAC address
+      # available in show version
+      # NOTE: the mac_address variable used in dhcp_provisioner role is
+      # different from this variable
+      # Expected format: xx:xx:xx:xx:xx:xx
+      system_mac_address: < str >
```

Comment:
we could support setting this as a regular inventory variable too.

Reply:
Rebased and pushed - system_mac_address now available at both places and test added to check precedence - please give feedback on the way it is documented.

PS: had to rebase and fix some broken import for errors

cf: c6bc86c